### ISSUE TITLE
Custom error messages

### ISSUE THREAD

**Issue by denkomanceski** [AndrewKeig/express-validation#61]
I want to use Joi's validate method to be able to add my custom validation message, because this is not very user friendly:
```
{
  "status": 400,
  "statusText": "Bad Request",
  "errors": [
    {
      "field": "pass",
      "location": "body",
      "messages": [
        "\"pass\" is required"
      ],
      "types": [
        "any.required"
      ]
    }
  ]
}
```
but Joi's validate message has 3 parameters: (value, schema, customMessage).. how do I achieve this ? I need to somehow inject value inside ?

**Comment by vinodkumarmarupu** [comment]
any solution?

**Comment by tesarm** [comment]
this could be done by modifying response:
https://github.com/AndrewKeig/express-validation#distinguish-errors-from-validationerrors

although it's not working currently - see https://github.com/AndrewKeig/express-validation/issues/67

**Comment by ankit7878** [comment]
it should give response in JSON instead of string.
express-validation give string in response . i think it may be because of "\" in response message. i dont know but how to get pure json instead of string in response ?

**Comment by AndrewKeig** [comment]
Closing, released 3.0
